pkg/volume/fc/fc_util.go

```diff
@@ -219,6 +219,10 @@ func (util *FCUtil) AttachDisk(b fcDiskMounter) (string, error) {
 	}
 	// mount it
 	globalPDPath := util.MakeGlobalPDName(*b.fcDisk)
+	if err := os.MkdirAll(globalPDPath, 0750); err != nil {
```

Comment:
I still prefer checking dir before mkdir. WatiForAttach calls AttachDisk many times, upfront mkdir in every call is not desirable

Reply:
@rootfs @jingxu97 

I checked the MkdirAll() source code and found an interesting thing.
MkdirAll() checks whether the directory exists at first and return nil if it exist immediately. I suppose the method is light weight.

But iscsi plugin already has same check on same method, I'm ok to check directory existence before create.

```
if err != nil && !os.IsNotExist(err) {
```

Reply:
ok, let keep it this way.